modules/nlu/src/backend/api.ts

```diff
@@ -15,6 +15,20 @@ import { IntentDefCreateSchema } from './validation'
 const SYNC_INTERVAL_MS = ms('5s')
 const USE_E1 = yn(process.env.USE_LEGACY_NLU)
 
+const removeSlotFromAllUtterances = (
```

Comment:
Please add it as import, not copy the actual code, I guess Visual Studio Code has "Quick fix" if this is not found.

Reply:
It comes from views, that for some reason are not available after build.
Also notice, that we have different function signature. It change all languages at once, not one like it was in original function.